### ISSUE TITLE
NoClassDefFoundError: com/google/common/io/Resources

### ISSUE THREAD

**Issue by rnorth** [testcontainers/testcontainers-java#398]
This is occurring in the tests for the newly separated mariadb module, using v1.4.0 of Testcontainers as the parent version. 

See https://travis-ci.org/testcontainers/testcontainers-java-module-mariadb#L847

I think we have an undeclared dependency on Guava inside `ContainerDatabaseDriver`:
```java
                URL resource = Resources.getResource(initScriptPath);
                String sql = Resources.toString(resource, Charsets.UTF_8);
```

Since we're trying to get rid of Guava because of its implications for dependency clashes, I'd say we should replace this with a fairly trivial non-Guava implementation.

**Comment by rnorth** [comment]
Just checked, the jdbc module has Guava as a provided dependency but it's no longer being shaded in, hence breakage.

**Comment by bsideup** [comment]
@rnorth 
Damn! I hate Maven >_<

Let's release it as 1.4.1 hotfix, will do the change

**Comment by bsideup** [comment]
Should be fixed in #399 

@rnorth can you try it (with jitpack) in MariaDB module?
